post/.gitignore

```diff
@@ -13,6 +13,7 @@
 
 # misc
 .DS_Store
+.vscode
 .env.local
 .env.development.local
 .env.test.local
```

Comment:
you can use an asterisk to avoid adding each env file manually
.env.* - should include every file whose name begins with .env